index.html

```diff
@@ -3,19 +3,60 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Karma</title>
+    <title> <img src="favicon.ico" alt="Karma logo">Karma</title>
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
     <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
-</head>
+</head> 
+
+<header>
+</header>
+
+
+<nav>
+        <img src="favicon.ico" alt="Karma logo">
+        <ul class="navbut">
+        
+            <li>Meet Karma</li>
+            <li>How it Works </li>
+            <li>Store</li>
+            <li>Blog</li>
+            <li>Help</li>
+            <li>Login</li>
+
+        </ul>
+    </nav>
 <body>
+    
+<section>
+    <img src="/img/first-background.jpg" alt="Person sat at a table with a tablet on it">
+    <div class="summary">
+        <h1>Introducing Karma</h1>
+        <h2>Bring Wifi with you, everywhere you go.</h2>     
+    </div>
+
+</section>
 
+<section>
+    <h1>Everyone needs a little Karma.</h1>
+        <ul class="imgpic">
+            <li><img src="/img/icon-devices.svg" alt="internet devices"></li>
```

Comment:
You need to change something in the image path here for something to appear on the page. 
You'll also need to add a width to these images in CSS, so the browser knows how big to make them - otherwise they may not appear at all as their width will be `0`.